### PR TITLE
Add LICENSE.md file

### PR DESCRIPTION
**Pull request by aucampia**
This license file contains the same content as preamble found in the
source files, however it also includes the following part which is
missing from the end (after "... CONTRACT, STRICT LIABILITY, OR TORT"):

> (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
> OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.

I'm somewhat of two minds if it is okay to include this additional part,
however I think it is given that the package uses "BSD" for the license
value in setuptools config and the
`License :: OSI Approved :: BSD License` classifier and most BSD licenses
end with same text after TORT, and this includes all BSD licenses on the
OSI site.

The license text itself is however not the same as the the 3-Clause BSD license
found at any of these locations:

- https://opensource.org/licenses/BSD-3-Clause
- https://spdx.org/licenses/BSD-3-Clause.html
- https://directory.fsf.org/wiki/License:BSD-3-Clause
  (differs from first two anyway)

The difference is mainly the first part of the third clause, which in the
isodate license is "Neither the name of the authors nor the names of its
contributors" instead of "Neither the name of the copyright holder nor the
names of its contributors may" (SDPX, OSI) and "The name of the author"
(FSF).

The most popular project with similar license text that I could find is
https://github.com/andialbrecht/sqlparse.

I formatted the file with:

```sh
npx prettier --prose-wrap always LICENSE.md
```

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -6,7 +6,8 @@ long_description_content_type = text/markdown
 url = https://github.com/isodate/isodate/
 author = Gerhard Weis
 author_email = gerhard.weis@proclos.com
-license = BSD
+license = BSD-3-Clause
```
Comment (aucampia) [inline setup.cfg:9]: `setup-cfg-fmt` changes license from `BSD` to `BSD-3-clause` technically this is closer to something accurate, as `BSD` is not a valid SPDX code. I'm okay with it, and `setup-cfg-fmt`'s behaviour here cannot really be changed anyway.

`setup-cfg-fmt` bases the `license` value on what it gets from https://pypi.org/project/identify/.

Comment (aucampia) [reply setup.cfg:9]:
```console
$ python3 -c 'from identify import identify; print(identify.license_id("LICENSE.md"))'
BSD-3-Clause
```

File: LICENSE.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+Copyright (c) 2021, Hugo van Kemenade and contributors Copyright (c) 2009-2018,
+Gerhard Weis and contributors Copyright 2009, Gerhard Weis All rights reserved.
```
Comment (hugovk) [inline LICENSE.md:2]: Maybe clearer on multiple lines?
```suggestion
Copyright (c) 2021, Hugo van Kemenade and contributors
Copyright (c) 2009-2018, Gerhard Weis and contributors
Copyright 2009, Gerhard Weis
All rights reserved.
```

**Comment by hugovk** [comment]
Diffing (after adjusting newlines):
```diff
--- bsd-3.txt	2021-11-06 21:34:55.000000000 +0200
+++ isotext.txt	2021-11-06 21:33:26.000000000 +0200
@@ -1,4 +1,6 @@
-Copyright <YEAR> <COPYRIGHT HOLDER>
+
+Copyright 2009, Gerhard Weis
+All rights reserved.
 
 Redistribution and use in source and binary forms, with or without
 modification, are permitted provided that the following conditions are met:
@@ -10,19 +12,17 @@
 this list of conditions and the following disclaimer in the documentation
 and/or other materials provided with the distribution.
 
-3. Neither the name of the copyright holder nor the names of its contributors
+3. Neither the name of the authors nor the names of its contributors
 may be used to endorse or promote products derived from this software
 without specific prior written permission.
 
 THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
 AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
 IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
-ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE
 LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
 CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
 SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
 INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
-CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
-ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF
-THE POSSIBILITY OF SUCH DAMAGE.
+CONTRACT, STRICT LIABILITY, OR TORT
 ```
The lack of full stop after `TORT` looks like it was originally a copy/paste mistake.

**Comment by hugovk** [comment]
https://github.com/gweis/isodate/pull/57 came to much the same conclusion as you.

File: LICENSE.md (hunk 2 above)
Comment (aucampia) [reply LICENSE.md:2]: Will fix

File: LICENSE.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2021, Hugo van Kemenade and contributors
+Copyright (c) 2009-2018, Gerhard Weis and contributors
+Copyright 2009, Gerhard Weis
```
Comment (hugovk) [inline LICENSE.md:3]: Ah, should probably have a `(c)` here too?
```suggestion
Copyright (c) 2009, Gerhard Weis
```

**Comment by hugovk** [comment]
Thank you!

**Pull request merged**